app/src/main/java/com/folen/androidshowreel/activity/FragmentActivity.java

```diff
@@ -0,0 +1,86 @@
+package com.folen.androidshowreel.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import com.folen.androidshowreel.R;
+import com.folen.androidshowreel.base.BaseActivity;
+import com.folen.androidshowreel.util.FirstFragment;
+import com.folen.androidshowreel.util.SecondFragment;
+
+public class FragmentActivity extends BaseActivity {
+
+    private Toolbar mToolbar;
+    private Fragment fragment;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_fragment);
+
+        setupToolbar();
+    }
+
+    private void setupToolbar() {
+        mToolbar = findViewById(R.id.fragment_toolbar);
+        setSupportActionBar(mToolbar);
+        mToolbar.setTitle("Fragments");
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.toolbar_menu, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
```

Comment:
Zrob taka konstrukcje tych fragmentow zeby zamiast tekstow Fragment1, 2, 3 itp wyswietlaly sie:

```
ADDED [timestamp]
REPLACED [timestamp]
```
Bo tak to first  fragment zawsze jest add, a second zawsze jest replace, a ja chcialem zebys ty zobaczyl roznice.